addons/point_of_sale/controllers/main.py

```diff
@@ -133,6 +133,13 @@ def invoice_request_screen(self, **kwargs):
                 else:
                     errors['generic'] = _("No sale order found.")
 
+        if kwargs.get('order_uuid'):
+            fetch_order_from_qr = request.env['pos.order'].sudo().search([('uuid', '=', kwargs.get('order_uuid'))], limit=1)
+        if not fetch_order_from_qr:
```

Comment:
We cannot check the condition of static variable outside the scope

Reply:
yup, I missed this
thanks for the detailed explanation :innocent: